src/shell.js

```diff
@@ -510,5 +510,3 @@ assert(!ENVIRONMENT_IS_SHELL, "shell environment detected but not enabled at bui
 #endif
 
 #endif // ASSERTIONS
-
-{{BODY}}
```

Comment:
this seems like a different and maybe not NFC change.  Can the user override this shell file?

Reply:
This change originates from PR #18398, so let me answer this question there.

Note that this PR was marked as draft since GitHub does not allow me to stack PRs when I use my own fork to base my branch on (instead of pushing the branches to the main repository). I could change this workflow next time, but I'm a little uncomfortable with pushing branches to the main repository as a volunteer external contributor.